src/main/java/duke/todo/TaskList.java

```diff
@@ -0,0 +1,225 @@
+package duke.todo;
+
+import duke.DukeException;
+
+import java.lang.StringBuilder;
+import java.util.List;
+import java.util.ArrayList;
+
+public class TaskList {
+    private List<Task> taskList;
+    private int counter;
```

Comment:
Could just use .size() method instead of having a counter?

Reply:
Good suggestion:) I did it in a rush and couldn't recall .size().